### PR TITLE
Added script/formatting/validator.py

### PR DESCRIPTION
**Pull request by akhileshsiddhanti**
validator.py identifies all print and possibly commented code in /src directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ git config core.hooksPath .githooks
 
 We invite you to help us build the future of visual data management DBMSs.
 
+```
```
Comment (jarulraj) [inline README.md]: This is likely not needed?

File: script/formatting/validator.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+# ==============================================
+# LOGGING CONFIGURATION
+# ==============================================
+
+EXIT_SUCCESS = 0
+EXIT_FAILURE = -1
+
+VALIDATOR_PATTERNS = [ re.compile(patterns) for patterns in [
+    r"print"
```
Comment (jarulraj) [inline script/formatting/validator.py]: Nice!

File: script/formatting/validator.py [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+EVA_TEST_DIR = os.path.join(EVA_DIR, "test")
+EVA_SCRIPT_DIR = os.path.join(EVA_DIR, "script")
+
+DIRS = [EVA_SRC_DIR,EVA_TEST_DIR]
+
+# ==============================================
+# UTILITY FUNCTION DEFINITIONS
+# ==============================================
+
+def comment_contains_code(line):
```
Comment (jarulraj) [inline script/formatting/validator.py]: rename as `contains_commented_out_code`?

File: script/formatting/validator.py [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+    if 'utf-8' in line:
+        return False
+
+    if not line.startswith('#'):
+        return False
+
+    line = line.lstrip(' \t\v\n#').strip()
+
+    regex_list = [
+        r"def .+\)[\s]*[->]*[\s]*[a-zA-Z_]*[a-zA-Z0-9_]*:$",
```
Comment (jarulraj) [inline script/formatting/validator.py]: Add a comment discussing the need for these regexes?

Comment (akhileshsiddhanti) [reply script/formatting/validator.py]: Added

File: script/formatting/validator.py (hunk 3 above)
Comment (akhileshsiddhanti) [reply script/formatting/validator.py]: Renamed.

File: script/formatting/validator.py (hunk 2 above)
Comment (akhileshsiddhanti) [reply script/formatting/validator.py]: Thank you!

File: README.md (hunk 1 above)
Comment (akhileshsiddhanti) [reply README.md]: Sorry, removed it.

**Pull request merged**